checkov/common/runners/runner_registry.py

```diff
@@ -283,3 +285,13 @@ def get_enriched_resources(repo_roots: List[Union[str, os.PathLike]]) -> Dict[st
                                 "skipped_checks": skipped_checks,
                             }
         return enriched_resources
+
+    @staticmethod
+    def strip_code_blocks_from_json(report_jsons):
+        for report in report_jsons:
+            results = report.get('results', [])
```

Comment:
it seems that at the next line you expect "results" us dict.
maybe you meant: 
`results = report.get('results', dict())`
?

Reply:
I think Itai is right 👍 but better to use
```python
results = report.get('results', {})
```